src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -1121,12 +1122,12 @@ private DebugId GetClosureId(SyntaxNode syntax, ArrayBuilder<ClosureDebugInfo> c
                 closureId = new DebugId(closureDebugInfo.Count, CompilationState.ModuleBuilderOpt.CurrentGenerationOrdinal);
             }
 
-            int syntaxOffset = _topLevelMethod.CalculateLocalSyntaxOffset(syntax.SpanStart, syntax.SyntaxTree);
+            int syntaxOffset = _topLevelMethod.CalculateLocalSyntaxOffset(syntax.SpanStart, syntax.SyntaxTree, localKind);
```

Comment:
>localKind [](start = 111, length = 9)

I haven't realized the method is used for calculating the offsets of lambdas and closures - it's my fault I should have renamed the method to reflect that. 

Since here we calculate offset for the closure itself and not for the closed over variable I'd just pass UserDefined. Or even better default(SynthesizedLocalKind), which is the same but it is easier to see that we don't have any variable (not even user var).

Alternatively, we could change the parameter to ```SynthesizedLocalKind?``` (nullable).